typings/index.d.ts

```diff
@@ -1219,6 +1245,8 @@ export class Interaction<Cached extends CacheType = CacheType> extends Base {
   public isContextMenu(): this is ContextMenuInteraction<Cached>;
   public isMessageComponent(): this is MessageComponentInteraction<Cached>;
   public isSelectMenu(): this is SelectMenuInteraction<Cached>;
+  public isModal(): this is ModalSubmitInteraction<Cached>;
+  public presentModal(modal: Modal | ModalOptions): Promise<void>;
```

Comment:
This method should be moved to subclasses, it doesn't exist on PING and MODAL interactions